applications/livetap/values.yaml

```diff
@@ -120,12 +120,33 @@ pg:
     # -- Affinity rules for the mock postgres pod
     affinity: {}
 
-tap_schema:
+tapSchema:
   image:
-    repository: {}
-    tag: "2.0.2"
+    # -- TAP schema image to ue. This must be overridden by each environment
+    # with the TAP schema for that environment.
+    repository: "lsstsqre/tap-schema-idfprod-tap"
```

Comment:
I wouldn't put this as a default as all the other values files should have something